src/common/image.c

```diff
@@ -1811,9 +1818,13 @@ int32_t dt_image_copy_rename(const int32_t imgid, const int32_t filmid, const gc
         sqlite3_step(stmt);
         sqlite3_finalize(stmt);
         DT_DEBUG_SQLITE3_PREPARE_V2(dt_database_get(darktable.db),
-                                    "INSERT INTO main.tagged_images (imgid, tagid)"
-                                    " SELECT ?1, tagid"
-                                    " FROM main.tagged_images"
+                                    "INSERT INTO main.tagged_images (imgid, tagid, position)"
+                                    " SELECT ?1, tagid, "
+                                    // 4294967295 << 32 = 0xFFFFFFFF00000000
```

Comment:
Looks like SQlite3 supports 0xFFFFFFFF00000000 directly have you tried to replace:
`
(4294967295 << 32)`
by
`0xFFFFFFFF00000000`

Maybe clearer to the reader.

Reply:
I'd read somewhere (probably not in the right place) that we could not do that. I haven't given a try....
Done.